api/PclusterApiHandler.py

```diff
@@ -162,13 +147,29 @@ def _wrapper_authenticated(*args, **kwargs):
 
     return _authenticated
 
+def get_scopes_list():
+  if not SCOPES_LIST:
+    return "openid"
+  elif "openid" not in SCOPES_LIST:
+    return SCOPES_LIST + " openid"
+  return SCOPES_LIST
 
+def get_redirect_url():
+  return f"{SITE_URL}/login"
+  
```

Comment:
nitpick(non-blocking):
thank you for providing this function.
I would use "redirect_uri" instead of redirect_url (also in the rest of the code),
to better comply with OAuth2 terminology (I don't think it's going to cause confusion)
but feel free to discard this comment since it's a very small nitpick